setup.cfg

```diff
@@ -68,10 +68,11 @@ install_requires =
     numpy==1.19.0; python_version=='3.6' and platform_machine!='loongarch64' and platform_python_implementation=='PyPy'
     numpy==1.20.0; python_version=='3.7' and platform_machine!='loongarch64' and platform_python_implementation=='PyPy'
     numpy==1.22.2; python_version=='3.8' and platform_machine!='loongarch64' and platform_python_implementation=='PyPy'
+    numpy==1.25.0; python_version=='3.9' and platform_machine!='loongarch64' and platform_python_implementation=='PyPy'
 
     # For Python versions which aren't yet officially supported,
     # we specify an unpinned Numpy which allows source distributions
     # to be used and allows wheels to be used as soon as they
     # become available.
     numpy; python_version>='3.12'
-    numpy; python_version>='3.9' and platform_python_implementation=='PyPy'
+    numpy; python_version>='3.10' and platform_python_implementation=='PyPy'
```

Comment:
Now that there's a PyPy 3.10, we might actually bump this to 3.11 directly (haven't tested the new PyPy yet, but generally compatibility has only gotten better, and 3.8/3.9 have been fully green in conda-forge for quite a while).

Reply:
Only when NumPy ships a `pypy310` wheel I'd say.